include/sqlpp11/detail/parse_date_time.h

```diff
@@ -27,102 +27,237 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <regex>
-
 namespace sqlpp
 {
   namespace detail
   {
-    // Parse a date string formatted as YYYY-MM-DD
-    //
-    inline bool parse_string_date(::sqlpp::chrono::day_point& value, const char* date_string)
+    namespace pdt
```

Comment:
`pdt` = acronym for parse date time?

I wonder if we really need a namespace in sqlpp::detail?

Reply:
Indeed `pdt` was an acronym for "parse date time". I added it because function names like `parse_unsigned` and `parse_character` sounded generic enough to me and I wanted to avoid possible name clashes in the future.

I removed the `pdt` namespace. I guess we can re-add it in the future if name clashes do happen.

Reply:
>  I guess we can re-add it in the future if name clashes do happen.

+1

Thanks!